src/main/java/net/sacredlabyrinth/phaed/simpleclans/Clan.java

```diff
@@ -277,6 +280,21 @@ public int getInactiveDays() {
         return (int) Math.floor(Dates.differenceInDays(new Timestamp(getLastUsed()), now));
     }
 
+    /**
+     * Returns the max number of days the clan has been inactive
```

Comment:
Returns the max number of days the clan can be inactive
A number <= 0 means it won't be purged